charts/kubeclarity/values.yaml

```diff
@@ -22,6 +22,33 @@ global:
 ## End of Global Values
 #######################################################################################
 
+#######################################################################################
+## Ingress Values
+
+ingress:
+  enabled: true
+  host: kubeclarity.example.com
+  className: ""
+## End of Ingress Values
+#######################################################################################
+
+#######################################################################################
+## Curl Values
+curl:
+  image:
+    registry: "docker.io"
+    repository: curlimages/curl
+    tag: 7.84.0
+## End of Curl Values
+#######################################################################################
+
+#######################################################################################
+## clusterRole Values
+clusterRole:
+  readClusterSecrets: false
+## End of clusterRole Values
+#######################################################################################
```

Comment:
Please move under `kubeclarity` section.
Also, please set the default value to `true`.